R/getGeneticInteractome.R

```diff
@@ -0,0 +1,62 @@
+# getGeneticInteractome.R
+
+#' Return BioGrid genetic interaction tags of physical interactors of the system
+#'
+#' @param mySys The dataframe of PPI between system's components
+#' @param key  a unique string key
+#' @param criterion A string, either "stringent" or "relaxed", specifying whether only GGI between physical interactors should be selected.
+#' @return The dataframe of system components and either their GGI between physical interactors should be selected (iff \code{criterion} == "stringent")
+#' or all GGI of physical interactors (iff \code{relaxed} == "stringent")
+#' @import xlsx
```

Comment:
Do you really need all of these `@import`'s for the function? The packages are available, since they are mentioned in `DESCRIPTION`: you should be able to use `<package>::<function>()`. I look at the code, and see that I would need to do a lot of searching until I find wher`bg()` comes from, just as one example.